static/js/actions/index.js

```diff
@@ -5,8 +5,13 @@ export const REQUEST_COURSE_LIST = 'REQUEST_COURSE_LIST';
 export const RECEIVE_COURSE_LIST_SUCCESS = 'RECEIVE_COURSE_LIST_SUCCESS';
 export const RECEIVE_COURSE_LIST_FAILURE = 'RECEIVE_COURSE_LIST_FAILURE';
 
+export const REQUEST_USER_PROFILE = 'REQUEST_USER_PROFILE';
+export const RECEIVE_USER_PROFILE_SUCCESS = 'RECEIVE_USER_PROFILE_SUCCESS';
+export const RECEIVE_USER_PROFILE_FAILURE = 'RECEIVE_USER_PROFILE_FAILURE';
```

Comment:
Can you add a CLEAR_PROFILE and related actions and reducers and the dispatch in the `componentWillUnmount`, similar to CLEAR_COURSE_LIST below?